homeassistant/components/sma/__init__.py

```diff
@@ -134,3 +138,13 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         data[PYSMA_REMOVE_LISTENER]()
 
     return unload_ok
+
+
+async def update_listener(hass, entry):
+    """Handle options update."""
+    _LOGGER.info("Change SMA language to: %s", entry.options.get(CONF_LANGUAGE))
```

Comment:
Do we need to log this as `info`? I think, `debug` would be more fitting.

https://developers.home-assistant.io/docs/development_guidelines/#log-messages

Reply:
Thanks fpr the hint. Is done.